script/kubernetes-scripts/setup-service-pod.bash

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+namespace=hmpps-accredited-programmes-preprod
```

Comment:
Can namespace be passed as a param when executing the bash script?

Reply:
Spot on - going to address this now. There is another bug in the script also, which I'm going to fix.